applications/StructuralMechanicsApplication/tests/cpp_tests/test_mpc_plate_tension.cpp

```diff
@@ -0,0 +1,175 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Yaman Zendaki , Hoang-Giang Bui
+//
+
+#include <iostream>
+#include "includes/gid_io.h"
```

Comment:
Can you comment this too?, will reduce the compilation time

Reply:
we will do this after the test is successful. There are some other commits preventing this one to merge.

Reply:
Yes, looks like is failing:

~~~sh
/home/travis/build/KratosMultiphysics/Kratos/applications//StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py
Check failed in file /usr/include/boost/numeric/ublas/storage.hpp at line 195:
i < size_
terminate called after throwing an instance of 'boost::numeric::ublas::bad_index'
  what():  bad index
~~~

Reply:
@yamanatoo can you trace down which line create it?

Reply:
done in c92174185cb83cfc3baf170babeb4a740567bc1c